server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -2403,7 +2427,7 @@ public Volume migrateVolume(MigrateVolumeCmd cmd) {
         if (diskOffering == null) {
             throw new CloudRuntimeException("volume '" + vol.getUuid() + "', has no diskoffering. Migration target cannot be checked.");
         }
-        if (!doesTargetStorageSupportDiskOffering(destPool, diskOffering)) {
+        if (VolumeTagsStoragePoolStrictness.valueIn(destPool.getDataCenterId()) && !doesTargetStorageSupportDiskOffering(destPool, diskOffering)) {
```

Comment:
I'm referring to the above change in the PR https://github.com/apache/cloudstack/pull/4257